logic.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from time import sleep
+
+
+import schedule
+
+database = {} # validator database 
```

Comment:
is it going to be a dict in memory or you want to introduce a real database here in the future?